electionleaflets/apps/api/tests/test_views.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from electionleaflets.apps.api.views import LeafletFilter
+from leaflets.models import Leaflet
+from django.utils import timezone
+from uk_political_parties.models import Party
```

Comment:
What are your thoughts on the future of this?

At the moment, the `uk_political_parties` is a bit of a pain point. The package repo itself is abandoned/archived. One of the problems it is causing us is we can't run `makemigrations --check` in CI because there is a missing migration in that package.
https://github.com/DemocracyClub/electionleaflets/blob/add000dd33eff87f9afcee484d57a3205d2154c0/.circleci/config.yml#L68-L69
I think I would like us to remove the dependency on that package and bring the handful of things we are using from it (I think just the party and emblem models) inline into this application.

If this one is something to address in a future PR, cool. Lets leave it for now.